cdk/src/pkg.ts

```diff
@@ -0,0 +1,11 @@
+import { access, readFile } from 'node:fs/promises'
+import { resolve } from 'node:path'
+
+const path = resolve('package.json')
```

Comment:
could possibly pick up `cdk/package.json`, revert to using `fileURLToPath(new URL('......`